### ISSUE TITLE
Propose format for hosting the CSER info pages/files on the AnVIL portal

### ISSUE THREAD

**Issue by NoopDog** [anvilproject/anvil-portal#2109]
#### Need 
Some consortia have no homepage. Additionally, some consortia with home pages will lose support for funding their home pages once the project is complete and need a new home for their consortia-specific content once their project funding has ended. It is desired by AnVIL stakeholders to use the AnVIL portal website as a place to hold detail pages for consortia. The CSER consortia is the first example of a consortium needing to re-host its information page since the project completing. 

#### Approach

- [ ] Meet with CSER members to understand their requirements and timeline.
- [ ] Review CSER materials provided for hosing.
- [ ] Create a UI mockup of the CSER home page and review it with stakeholders.

#### Definition of Done
1. We have an approved UI mock for including CSER information on the AnVIL portal.

**Comment by NoopDog** [comment]
Superseded by 2135
